conda/utils.py

```diff
@@ -292,6 +292,20 @@ def get_yaml():
     return yaml
 
 
+# Restores YAML 1.1 boolean flexibility.
+yaml_bool_ = {
+    'true': True, 'yes': True, 'on': True,
+    'false': False, 'no': False, 'off': False
+}
+def yaml_bool(s, passthrough=None):
+    if type(s) is bool:
```

Comment:
I've always thought this was more semantic, but for whatever reason I've also always seen people say prefer `isinstance`.  Works for me for now though.